flocker/node/script.py

```diff
@@ -76,6 +80,46 @@ def _get_external_ip(host, port):
         sock.close()
 
 
+def configuration_from_options(options):
+    """
+    Extract configuration from provided options.
+
+    :param dict options: Dictionary containing command line options.
+
+    :return dict: Dictionary containing the desired configuration.
+    """
+    control_service_options_file = options[u'control-service-config']
+
```

Comment:
The backend plugins can supply schema additions so I think it works.

I'd really like to see this done with jsonschema.